CMakeLists.txt

```diff
@@ -33,7 +33,7 @@ option(STATIC_LINK "link statically against dependencies" OFF)
 option(BUILD_SHARED_LIBS "build lokinet libraries as shared libraries instead of static" ON)
 option(SHADOW "use shadow testing framework. linux only" OFF)
 option(XSAN "use sanitiser, if your system has it (requires -DCMAKE_BUILD_TYPE=Debug)" OFF)
-option(JEMALLOC "use jemalloc. Not required on BSD" OFF)
+option(WITH_JEMALLOC "use jemalloc as allocator" OFF)
 option(TESTNET "testnet build" OFF)
```

Comment:
`jemalloc` could easily be its own commit or even PR

Reply:
yeah probably